fuzzyfinder.go

```diff
@@ -553,7 +618,6 @@ func (f *finder) find(slice interface{}, itemFunc func(i int) string, opts []Opt
 	if err := f.initFinder(items, matched, opt); err != nil {
 		return nil, errors.Wrap(err, "failed to initialize the fuzzy finder")
 	}
-	defer f.term.close()
```

Comment:
Should we call `f.term.Screen().Fini()` at the end?

Reply:
I call `Fini()` [here](https://github.com/ktr0731/go-fuzzyfinder/pull/32/files?file-filters%5B%5D=.go#diff-902eb73b67dfd52495aa77c7330f4734c5617b00a9faaebb99c97ec583a927c3R706) and [here](https://github.com/ktr0731/go-fuzzyfinder/pull/32/files?file-filters%5B%5D=.go#diff-902eb73b67dfd52495aa77c7330f4734c5617b00a9faaebb99c97ec583a927c3R726)

Reply:
I was misunderstanding, as you say, I can call `Fini()` in `find`